org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -22,7 +22,6 @@
 import java.util.function.Consumer;
 
 import org.eclipse.core.runtime.CoreException;
```

Comment:
agree. Here this import is needed for calculating classpath. i'll move the logic to language server, and then delete the import